### PR TITLE
Staff Check In

### PR DESCRIPTION
**Pull request by StoneT2000**
- Added checkbox to check off if you are staff
- box visible only to STAFF members

### CODE REVIEW

File: src/actions/eventsActions.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const checkIn = (attendanceCode) => async dispatch => {
         'Content-Type': 'application/json',
         Authorization: `Bearer ${Storage.get('token')}`,
       },
-      body: JSON.stringify({ attendanceCode }),
+      body: JSON.stringify({ attendanceCode: info.attendanceCode, asStaff: info.asStaff }),
```
Comment (sumeet-bansal) [inline src/actions/eventsActions.js]: maybe use a [`pick`](https://underscorejs.org/#pick) function here? e.g. `JSON.stringify(pick(info, ['attendanceCode', 'asStaff']))`

File: src/components/EventCheckIn/index.jsx [hunk 2]
```diff
@@ -20,7 +23,14 @@ const EventCheck = props => {
...
           Submit
         </Button>
+
       </div>
+      { props.user.profile.accountType === "STAFF" &&
+        <div className="staff-checkin-div">
+          <label className="staff-check-in-label">Staff Check In</label>
+          <Checkbox onChange={props.onAsStaffChange}/>
+        </div>
+      }
```
Comment (sumeet-bansal) [inline src/components/EventCheckIn/index.jsx]: not sure if the current format is convention but I think this block would be a little more obvious/readable as a conditional

Comment (StoneT2000) [reply src/components/EventCheckIn/index.jsx]: This is ok, would use a conditional if there was something to display if accountType wasn't STAFF but there isn't anything to display.

File: src/actions/eventsActions.js (hunk 1 above)
Comment (StoneT2000) [reply src/actions/eventsActions.js]: I think this works fine, using pick means we would have to import another library when there's no serious need

**Pull request merged**